src/object/wind.cpp

```diff
@@ -47,6 +49,8 @@ Wind::Wind(const ReaderMapping& reader) :
 
   reader.get("acceleration", acceleration, 100.0f);
 
+  reader.get("affectsbadguys", affects_badguys, false);
```

Comment:
Can we name the lisp tag affects-badguys (we separate words with a dash in lisp by default).

Reply:
Updated as suggested.